src/js/view/controls/components/menu/menu.js

```diff
@@ -173,6 +176,51 @@ export default class Menu extends Events {
     }
     createTopbar() {
         const topbar = createElement(`<div class="jw-submenu-topbar"></div>`);
+        const itemsContainer = this.itemsContainer.el;
+        const settingsMenu = this.mainMenu;
+        const categoryButton = this.categoryButton;
+        this.topbarUI = new UI(topbar).on('keydown', (evt) => {
```

Comment:
should we also `stopPropagation` here to avoid any conflicts with the internal menu listeners for focus or is that not needed?

Reply:
Nope, but I should rename that function to make it clear, so good comment either way :)